core/client/app/controllers/settings/connections/edit.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+
+const {Controller} = Ember;
+
+export default Controller.extend({
+
+isEditing: true,
+trustedDomains: [],
+
+actions: {
+    addUrl(url){
```

Comment:
connections/edit & connections/new controllers have same actions. Eliminate duplication of code. Create a mixin and include the mixin in both controllers.